api/v1alpha1/hardwareclassification_types.go

```diff
@@ -31,7 +31,7 @@ type HardwareClassificationSpec struct {
 	HardwareCharacteristics HardwareCharacteristics `json:"hardwareCharacteristics,omitemty"`
```

Comment:
you have a typo here
```suggestion
	HardwareCharacteristics HardwareCharacteristics `json:"hardwareCharacteristics,omitempty"`
```